### DISCUSSION TITLE
Multiple Usage Policies

### DISCUSSION THREAD

**Discussion by aitorcelaya** [International-Data-Spaces-Association/DataspaceConnector#371] [Q&A]
We are looking at the implementation of Usage Policies in the Dataspace Connector v7.1.0 and it looks like currently we can only attach a single policy.

Is there any plan to implement multiple Usage Policies for the upcoming versions? 

i.e.
1. Usage Policy to only allow X Connector(s) access.
2. Usage Policy to only allow information to be accessed for Y time range.

**Comment by omarsilva1** [comment]
Hello aitorcelaya
It is possible to add multiple usage policies to a resource, one policy per rule. If one of the policies aren't valid, access is denied.
You can restrict policy access to one connector, I dont believe this works for multiple connectors. If this is requested you can open an issue.

There are two time limited usage policies: usage duration and time interval, examples and more explanation are shown [here](https://international-data-spaces-association.github.io/DataspaceConnector/Documentation/v6/UsageControl).
